plugins/wrangler/api_token.go

```diff
@@ -0,0 +1,52 @@
+package wrangler
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIToken,
+		DocsURL:       sdk.URL("https://developers.cloudflare.com/workers/wrangler/system-environment-variables/"),
```

Comment:
I think this docs page would make more sense for the Cloudflare API token:
```suggestion
		DocsURL:       sdk.URL("https://developers.cloudflare.com/fundamentals/api/get-started/create-token/"),
```